index.js

```diff
@@ -165,23 +168,23 @@ class GitlabScm extends Scm {
             'Invalid config for Gitlab'
         );
 
-        const gitlabConfig = {};
-
-        if (this.config.gitlabHost) {
-            gitlabConfig.host = this.config.gitlabHost;
-            gitlabConfig.protocol = this.config.gitlabProtocol;
-            gitlabConfig.pathPrefix = '';
-        }
+        this.gitlabConfig = { prefixUrl: `${this.config.gitlabProtocol}://${this.config.gitlabHost}/api/v4` };
 
-        this.breaker = new Breaker(Request, this.config.fusebox);
+        // eslint-disable-next-line no-underscore-dangle
+        this.breaker = new Breaker(this._gitlabCommand.bind(this), {
```

Comment:
why use breaker? Why not use the native retry capabilities of the library https://github.com/sindresorhus/got/blob/main/documentation/7-retry.md

Reply:
It was my understanding that circuit breaker vs retry are not exactly the same thing:
- https://dev.to/supriyasrivatsa/retry-vs-circuit-breaker-346o
- https://docs.microsoft.com/en-us/azure/architecture/patterns/circuit-breaker
- https://dantt.medium.com/circuit-breaker-and-retry-64830e71d0f6

It might be good to look for replacements or upgrades to the circuit-fuses library, but I think that is out of scope for [this issue](https://github.com/screwdriver-cd/screwdriver/issues/1961). Also, I'd rather not change two major packages at the same time.